src/cli/subcommands/healthcheck_cmd.rs

```diff
@@ -76,9 +76,31 @@ impl HealthcheckCommand {
         );
 
         for _ in ticker {
-            let response = reqwest::get(&url).await?;
-            let status = response.status();
-            let text = response.text().await?;
+            let (status, text) = {
+                match reqwest::get(&url).await {
+                    Ok(response) => {
+                        let status = response.status();
+                        let text = match response.text().await {
+                            Ok(t) => t,
+                            Err(e) => {
```

Comment:
nit: it'd be a bit more expressive with `Err(e) if !wait`

Reply:
Done.